diag_manager/fms_diag_file_object.F90

```diff
@@ -788,6 +824,132 @@ subroutine open_diag_file(this, time_step, file_is_opened)
   diag_file => null()
 end subroutine open_diag_file
 
+!> \brief Write the time metadata to the diag file
+subroutine write_time_metadata(this)
+  class(fmsDiagFileContainer_type), intent(inout), target :: this !< The file object
+
+  class(fmsDiagFile_type), pointer  :: diag_file      !< Diag_file object to open
+  class(FmsNetcdfFile_t),  pointer  :: fileobj        !< The fileobj to write to
+  character(len=50)                 :: time_units_str !< Time units written as a string
+  character(len=50)                 :: calendar       !< The calendar name
+
+  character(len=:), allocatable :: time_var_name !< The name of the time variable as it is defined in the yaml
+
+  diag_file => this%FMS_diag_file
+  fileobj => diag_file%fileobj
+
+  time_var_name = diag_file%get_file_unlimdim()
+  call register_axis(fileobj, time_var_name, unlimited)
```

Comment:
I may be missing this, but where is the variable `unlimited` coming from?

Reply:
`unlimited` comes fro fms2_io_mod